packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1687,8 +1688,31 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 summaryLogger,
             });
 
+            const summary = summarizeResult.summary.tree;
+            const summaryTree = summary[channelsTreeName] as ISummaryTree;
```

Comment:
This cast is dangerous, because of: `this.disableIsolatedChannels`. It needs to be backwards compatible, or you need to move the counting into `summarizeInternal()` or lower.